godtools/Views/Cells/HomeToolTableViewCell.swift

```diff
@@ -76,7 +76,7 @@ class HomeToolTableViewCell: UITableViewCell {
         titleLabel.isEnabled = isAvailableInPrimaryLanguage
         titleLabel.text = resource.localizedName(language: primaryLanguage)
         
-        languageLabel.text = resource.isAvailableInLanguage(parallelLanguage) ? parallelLanguage!.localizedName() : nil
+        languageLabel.text = resource.isAvailableInLanguage(parallelLanguage) ? parallelLanguage!.localizedName : nil
```

Comment:
I think this is more of a specialized case. Here we want to set the parallel language name only if there is a translation for that resource. We could create a method for that, but I don't think that logic is used elsewhere.